CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 2.3.1-dev
+## 2.3.1
+
+* Switch to using package:lints.
```

Comment:
Fwiw we typically don't include this type of change in the changelog. It doesn't really hurt but it is probably just noise to consumers of the package.

Reply:
I'm mostly concerned with capturing the `repository` field change w/ a publish. It's a light patch release, so I included descriptions of the other changes in here as well.

Reply:
It is an implementation detail of the package (does not affect consumers), and the changelog is generally for users. But, I don't feel strongly or anything about removing it, you can leave it in if you like.

Longer changelogs aren't better, if the info isn't useful :).